### ISSUE TITLE
Do IPFS links need to be supported in the new Markdown viewer?

### ISSUE THREAD

**Issue by burtonator** [hicommonwealth/commonwealth#9240]
## Describe the bug

Right now we support this within the quill renderer:

```typescript
markdownRenderer.image = (href, title, text) => {
  if (href?.startsWith('ipfs://')) {
    const hash = href.split('ipfs://')[1];
    if (hash) {
      href = `https://ipfs.io/ipfs/${hash}`;
    }
  }
  return `<img alt="${text}" src="${href}"/>`;
};
```

This fixes all img URLs so they load from https://ipfs.io not the ipfs:// scheme.

Is this used? I'll have to patch the img plugin in MDXEditor/Lexical to support this.  I think it shouldn't be too hard to implement though.

### Initial conditions

**Environment:** 

**Branch/Release version:** 

**Browser:** 

**Wallet:**

**Comment by burtonator** [comment]
@timolegros yeah. I agree with this. Though technically someone can link to an external image manually if/when we enable direct markdown mode.  They could just paste/type the image URL directly into a markdown image reference.

Right now it's not possible though.

I'd like to NOT enable this too as it will be a big hairy and require me to have another patch for the img plugin for MDXEditor and that might not be accepted.

**Comment by dillchen** [comment]
All good, don't need

Sent via Superhuman ***@***.***>
